opentech/apply/review/models.py

```diff
@@ -100,24 +101,16 @@ def __str__(self):
     def __repr__(self):
         return f'<{self.__class__.__name__}: {str(self.form_data)}>'
 
-    def data_and_fields(self):
-        for stream_value in self.form_fields:
-            try:
-                data = self.form_data[stream_value.id]
-            except KeyError:
-                pass  # It was a named field or a paragraph
-            else:
-                yield data, stream_value
-
     @property
-    def fields(self):
-        return [
-            field.render(context={'data': data})
-            for data, field in self.data_and_fields()
-        ]
-
-    def render_answers(self):
-        return mark_safe(''.join(self.fields))
+    def for_latest(self):
+        return self.revision == self.submission.live_revision
+
+    def get_compare_url(self):
```

Comment:
This is another one of those cross-over moments.
Can this be renamed to be more verbose? again, this is the submission compare url, not the review one and it reads   as the latter